deploy/keycloak/keycloak-init.sh

```
@@ -25,6 +25,7 @@ kubectl create ns $NS || true
 echo "mosip-signup-client secret  is created already"
 SIGNUP_CLIENT_SECRET_KEY='mosip_signup_client_secret'
 SIGNUP_CLIENT_SECRET_VALUE=$(kubectl -n keycloak get secrets keycloak-client-secrets -o jsonpath={.data.$SIGNUP_CLIENT_SECRET_KEY} | base64 -d)
+IAMHOST_URL=$(kubectl -n signup get cm esignet-global -o jsonpath={.data.mosip-iam-external-host})
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)